README.md

```diff
@@ -32,11 +39,75 @@ efficient and structured communication between different software components. By
 solely for protobufs, the RobôCIn team ensures consistency, version control, and easy access to shared data structures
 across various projects within the organization.
 
-<a name="build"></a>
+Protocol Buffers offer numerous advantages, such as efficient serialization, platform independence, and language
+interoperability. By using Protocol Buffers, you can easily serialize, deserialize, and exchange structured data between
+different systems or languages.
+
+To work with the protocols in this repository, you will need to have Protocol Buffers installed in your development
+environment. You can visit the [Protocol Buffers GitHub repository](https://github.com/protocolbuffers/protobuf) for
+installation instructions specific to your programming language.
+
+<a name="install"></a>
+
+## Installation
+
+> **Note**: If you intend to utilize this repository on an alternative operating system, please don't hesitate to
+> raise an issue. We are committed to providing support and helping you modify the installation process to suit your
```

Comment:
Shouldn't it be "and help you modify..."? Or maybe "providing support by helping you modify..."